lib/live_sup_web/live/widgets/blameless/blameless_current_incidents_live.ex

```diff
@@ -196,4 +95,99 @@ defmodule LiveSupWeb.Live.Widgets.Blameless.CurrentIncidentsLive do
 
   def commander_title(%{title: ""}), do: ""
   def commander_title(%{title: title}), do: "(#{title})"
+
+  def no_incidents_icon(assigns),
```

Comment:
You can remove the `assigns`  and also the `~H` since this is not a component, it's just a regular function.

Reply:
Also, since you are using this on several widgets. You could just create a component [here](https://github.com/livesup-dev/livesup/tree/main/lib/live_sup/views), take a look to [this component](https://github.com/livesup-dev/livesup/blob/main/lib/live_sup_web/live/home/components/favorite_row_component.ex).